### PR TITLE
Track complex queries

### PR DESCRIPTION
**Pull request by Vegter**

### CODE REVIEW

File: src/gobapi/profiled_query.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import time
+
+from sqlalchemy import event
+from sqlalchemy.engine import Engine
+
+
+class ProfiledQuery():
+
+    LONG_DURATION = 60 * 10  # 10 minutes or longer
```
Comment (JKoetsier) [inline src/gobapi/profiled_query.py]: Maybe start with a slightly higher value and work our way down? Say 30 minutes? Many queries take 10 minutes at the moment.

File: src/gobapi/storage.py [hunk 2]
```diff
@@ -59,6 +61,7 @@ def connect():
     metadata = MetaData(engine)
 
     set_session(session)
+    profiled_query.activate()
```
Comment (JKoetsier) [inline src/gobapi/storage.py]: Nice!

File: src/gobapi/profiled_query.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        conn.info.setdefault('query_info', []).append(profiled_query)
+
+    @event.listens_for(Engine, "after_cursor_execute")
+    def after_cursor_execute(conn, cursor, statement,
+                             parameters, context, executemany):
+        profiled_query = conn.info['query_info'].pop(-1)
+        profiled_query.set_end()
+        if profiled_query.is_complex:
+            print("WARNING: Complex query detected")
+            print(profiled_query)
```
Comment (JKoetsier) [inline src/gobapi/profiled_query.py]: Is it an idea to remove new lines from the query? This way it's much easier to get the queries from Kibana (do some auto formatting after), as multiline logs in Kibana are a bit sh*t

File: src/gobapi/profiled_query.py (hunk 1 above)
Comment (Vegter) [reply src/gobapi/profiled_query.py]: Set to 30

File: src/gobapi/profiled_query.py (hunk 3 above)
Comment (Vegter) [reply src/gobapi/profiled_query.py]: Done

**Pull request merged**